examples/bulb/js/dan2.js

```diff
@@ -131,19 +131,21 @@ var dan2 = (function () {
         _i_chans = new ChannelPool();
         _o_chans = new ChannelPool();
 
-        function on_failure () {
+        function on_failure (err) {
```

Comment:
I'm not responsible for this bug.
Referenced [paho mqtt doc](http://www.eclipse.org/paho/files/jsdoc/symbols/Paho.MQTT.Client.html#connect) and the context of code, I guess [620a9d: Try to fix parameters inconsistency](https://github.com/IoTtalk/iottalk-js/pull/2/commits/620a9d5bbf54b8eec3f85bc2fd545dea32280d04) can solve this.